web/src/components/ChatContainer.tsx

```diff
@@ -61,16 +64,25 @@ type ChatContainerProps = {
 }
 
 const ChatContainer = ({ city, language }: ChatContainerProps): ReactElement => {
-  const { t } = useTranslation('chat')
-  const [chatVisible, setChatVisible] = useState(false)
+  const [searchParams, setSearchParams] = useSearchParams()
```

Comment:
🙃 I know they're technically called search params but can we call the state here query params instead, just for future me? So, `const [queryParams, setQueryParams] = useSearchParams()`